src/extension.ts

```diff
@@ -212,6 +212,10 @@ function getScopeLevel(configurationKey : string, key : string) : ScopeInfo{
 
 function getIndentationRules(): LanguageConfiguration {
   return {
```

Comment:
please add a comment linking to the source of the regexps

Reply:
I added the comment. How is it?